### PR TITLE
MainNet 1.5.2 protocol upgrade

### PR DESCRIPTION
**Pull request by sacherjj**
Activation Era 10552
UTC: 8/29/2023 13:04:16

### CODE REVIEW

File: config/config-example.toml [hunk 1]
```diff
@@ -31,6 +53,43 @@ abbreviate_modules = false
...
+
+
+# =======================================
+# Configuration options for Zug consensus
+# =======================================
+[consensus.zug]
+
+# Request the latest protocol state from a random peer periodically, with this interval.
+# '0sec' means it is disabled and we never request the protocol state from a peer.
+sync_state_interval = '1sec'
```
Comment (cspramit) [inline config/config-example.toml:68]: should not this be 0, disabled

Comment (cspramit) [reply config/config-example.toml:68]: 2023-08-29 11:05:44.824000+00:00 isnt this the time for Era 10552

Comment (sacherjj) [reply config/config-example.toml:68]: Correct, I had the end Era instead of the start era and was off by one.  Corrected.

**Comment by sacherjj** [comment]
Went through known_addresses and pruned missing nodes.

**Review by cspramit — APPROVED** [review]

**Comment by cspramit** [comment]
Lgtm

**Review by cspramit — APPROVED** [review]

**Pull request merged**